hawkbit-runtime/docker/generator/template/Dockerfile-postgresql

```diff
@@ -0,0 +1,10 @@
+FROM hawkbit/hawkbit-update-server:{{HAWKBIT_VERSION}}
+
+ENV POSTGRESQL_DRIVER_VERSION={{POSTGRESQL_DRIVER_VERSION}}
```

Comment:
This could be removed now since you already added the driver to the runtime

Reply:
Whole file?

Reply:
Didn't understand what should I remove.